### ISSUE TITLE
Where is the original source of the badge?

### ISSUE THREAD

**Issue by inson1** [Kunzisoft/Github-badge#1]
@J-Jamet

**Comment by J-Jamet** [comment]
I don't understand the question.

**Comment by inson1** [comment]
@J-Jamet oh so did you create it?

**Comment by inson1** [comment]
If yes Why there isnt any license? Please?

**Comment by J-Jamet** [comment]
~Yes I created the SVG and PNG, you can use them under Creative Commons license v4 https://creativecommons.org/licenses/list.en~

**Comment by inson1** [comment]
@J-Jamet What if you added the license in the repository? It would be better :)
Thanks. I didnt know, where did they came from and you didnt have any star, but lots of poeple are using it, so I didnt expect it :) Great work

**Comment by zhanghai** [comment]
The SVG file content is saying CC-BY-SA 3.0 though.

And if I embed an image tag linking to the image here, do I need to (or how do I) give attribution for (CC-BY-*)?

**Comment by zhanghai** [comment]
nvm, IANAL but I don't think I'll "Distribute, or Publicly Perform the Work" by putting the original link in markdown. The viewers will also be getting it directly from the original source.

**Comment by J-Jamet** [comment]
Oh dear, you're right, I couldn't remember the source. I was sure I'd created them but they're actually images from andOTP, so sorry. So you'll have to ask the person concerned directly. Since the repository's license is MIT, I shouldn't have bothered.
https://github.com/andOTP/andOTP

**Comment by J-Jamet** [comment]
I will write a correct license and credit the author correctly. Sorry again

**Comment by inson1** [comment]
@flocke Hi, where did you get the badges? :) Did you create them?

**Comment by flocke** [comment]
Yes, I created them based on the SVG files of the F-Droid badges here: https://gitlab.com/fdroid/artwork/-/tree/master/badge/src

**Comment by mak448a** [comment]
Hey @flocke, is it enough to have you credited in my README file and include the badge?
